NEWS

```diff
@@ -1564,3 +1564,6 @@ version 2.3.1 (2020-12-14)
 when using an EB prior for mu
 - lower minimum tolerance `mintol_mu` (1e-5 instead of 1e-3) as a default value
 in `.BASiCS_MCMC_ExtraArgs`
+
+version 2.3.2 (2020-04-14)
```

Comment:
Same here. Date to be updated.

Reply:
lol definitely living in 2020 still

Reply:
time has a weird dimension these days ...